pkg/log/formatter.go

```diff
@@ -0,0 +1,76 @@
+package log
+
+import (
+	"fmt"
+	"github.com/applike/gosoline/pkg/encoding/json"
+	"github.com/fatih/color"
+	"strings"
+)
+
+type Formatter func(timestamp string, level int, format string, args []interface{}, err error, data Data) ([]byte, error)
+
+var formatters = map[string]Formatter{
+	"console": FormatterConsole,
+	"json":    FormatterJson,
+}
+
+func FormatterConsole(timestamp string, level int, format string, args []interface{}, err error, data Data) ([]byte, error) {
+	fieldString := getFieldsAsString(data.Fields)
+	contextString := getFieldsAsString(data.ContextFields)
+
+	levelStr := fmt.Sprintf("%-7v", LevelName(level))
```

Comment:
Why do you need to use `%v` here instead of `%d`?

Reply:
Do you mean `%s`? I think `LevelName(level)` returns a string, not a number. But yeah, I guess that could be done like for the channel in the next line

Reply:
switched to `%s`